jovian/jovian_nb_ext/main.js

```diff
@@ -38,9 +38,22 @@ define([
           const capture_env = window.jvn_params.capture_env;
           const create_new = window.jvn_params.create_new;
           const env_type = window.jvn_params.env_type;
-          const files = window.jvn_params.files;
+          const files =
+            "[" +
+            window.jvn_params.files
+              .split(",")
+              .map(e => "'" + e.trim() + "'")
+              .join(",") +
+            "]";
```

Comment:
Since the pattern is same for both files and artifacts.You can return it from a function.

Reply:
Sure, I will change to create a function to do that.